.gitlab-ci.yml

```diff
@@ -585,7 +585,10 @@ publish_community_operators:
   # needs:
   #   - "submit_preflight_redhat_image"
   tags: [ "runner:docker", "size:large" ]
-  image: $JOB_DOCKER_IMAGE
+  image: $JOB_DOCKER_IMAGE # TODO: change to a new image that includes dd-octo-sts
```

Comment:
yes, you'll have to use an image with dd-octo-sts or install though I don't know if that can work and is probably inefficient